test/integration/peers.integration.js

```diff
@@ -43,11 +43,13 @@ var monitorWSClient = {
 	autoReconnect: true,
 	query: {
```

Comment:
Might be nice to use it here the same function `generatePeerHeaders()` as other situation while generating WS client headers. Then overwrite only port property with 9999.

Reply:
@MaciejBaj Currently generatePeerHeaders are part of test/node which is registering global hooks. If i use that file in integration tests, then starting up default instance will become necessary. Do you think we should add this dependency to this integration tests?